src/functions.js

```diff
@@ -105,6 +105,14 @@ function filter(text) {
       {pattern: /(\\title{)(.*?)(})/g, newValue: '$2'},
       {pattern: /(\\usepackage\[(.*?)]{)(.*?)(})/g, newValue: ''},
       {pattern: /(\\usepackage{)(.*?)(})/g, newValue: ''},
+      {pattern: /(\\emph{)(.*?)(})/g, newValue: '$2'},
+      {pattern: /(\\acs{)(.*?)(})/g, newValue: '$2'},
+      {pattern: /(\\ac{)(.*?)(})/g, newValue: '$2'},
+      {pattern: /(\\acf{)(.*?)(})/g, newValue: '$2'},         
+      {pattern: /(\\paragraph{)(.*?)(})/g, newValue: '$2'},      
+      {pattern: /(\\subsubsection{)(.*?)(})/g, newValue: '$2'},      
+      {pattern: /(\$)(.*?)(\$)/g, newValue: '$2'},  // remove inline math tags          
```

Comment:
Can you give me an example of what this should do?

Reply:
Not OP, but this pattern removes inline math tags such as:

```latex
This is a sentence using an inline math tag $y = ab + x$. --> This is a sentence using an inline math tag .
```